vpr/src/pack/appack_context.h

```diff
@@ -122,6 +130,15 @@ struct APPackContext : public Context {
                                                 logical_block_types,
                                                 device_grid);
         }
+
+        // By default, when unrelated clustering is on, search for unrelated molecules
+        // that are within 1 tile from the centroid of the cluster.
+        // NOTE: Molecules within the same tile as the centroid are considered to have
```

Comment:
Comment should probably refer to 1 tile less, and just say what this controls (e.g. if set to 1 tile ...).
Then can comment we default to a low value (1.0) and it can grow if the clustering fails due to too many tiles required.